CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
-- Preview width
+- Image preview width when there's only one item.
+- Always re-fetch content when opening a card.
```

Comment:
Why?

Reply:
I made this change because caching the content (like we do) makes the user having to refresh the whole page when they want to get the latest content for a component they've already opened. Although this isn't the best solution, it provides more confidence at the cost of waiting for a refetch.

I'm open to discuss alternatives for this, tho.

Reply:
It makes sense to me as well! I was just wondering if it had anything to do with the other changes in the PR.

But what about the following scenario?
1. User A navigates to page 1;
2. User B changes the active configuration of block X from page 1;
3. User A decides to edit the block X.

The new changes will be fetched, but the Runtime will still be at the old state, right? Perhaps we should also update it after fetching the configurations.